src/go/k8s/tests/e2e/regressions/01-call-controller-status.yaml

```diff
@@ -0,0 +1,90 @@
+            - -c
+          args:
+            - |
+              set -xe
+              # using the same value found in the duck-tape test https://github.com/redpanda-data/redpanda/pull/9737/commits/cf98f44141a5b07eddf5d2f7e69868585c7246f4
+              MAX_INDEX=2000
+              CREATE_INDEX_LOWER_BOUND=500
+              FAILED_MSGS=""
+              USER_NAME="a_user"
+              USER_PASS="my_password"
+              USER_ALGO="SCRAM-SHA-256"
+              
+              ADMIN_API_PATH="http://cluster-0.cluster.$POD_NAMESPACE.svc.cluster.local:9644"
+              CONTROLLER_STATUS_PATH="v1/debug/controller_status"
+              USER_PATH="v1/security/users"
+              
+              # wait for admin server to be ready, should be serving quickly
+              CURL_CONTROLLER_STATUS_CMD="curl -so /dev/null -w "%{http_code}" ${ADMIN_API_PATH}/${CONTROLLER_STATUS_PATH}"
+              until [ "${status:-}" = "200" ]; do
+                status=$(${CURL_CONTROLLER_STATUS_CMD})
+                sleep 0.5
+              done
+              
+              # POST user information several hundred times and verify this is causing the number a huge number of committed indices
+              for i in $(seq $CREATE_INDEX_LOWER_BOUND); do
+                curl -s -XPOST -H 'Content-Type: application/json' -d '{"username":"'${USER_NAME}'","password":"'${USER_PASS}'","algorithm":"'${USER_ALGO}'"}' ${ADMIN_API_PATH}/${USER_PATH}
+              done              
+              
+              post_index=$(curl -s ${ADMIN_API_PATH}/${CONTROLLER_STATUS_PATH} | jq -r .commited_index)
+              
+              # we will not fail here, since this isn't fatal
+              if [ "${post_index}" -ge "${CREATE_INDEX_LOWER_BOUND}" ]; then
+                FAILED_MSGS="${FAILED_MSGS} POST: index is ${post_index}, which is more than ${CREATE_INDEX_LOWER_BOUND}"
+                echo "POST: index is ${post_index}, which is more than ${CREATE_INDEX_LOWER_BOUND}"
+              fi
+              
+              # PUT user information several hundred times and verify this is not causing a huge number of committed indices
+              for i in $(seq $CREATE_INDEX_LOWER_BOUND); do
+                curl -s -XPUT -H 'Content-Type: application/json' -d '{"password":"'${USER_PASS}'","algorithm":"'${USER_ALGO}'"}' ${ADMIN_API_PATH}/${USER_PATH}/${USER_NAME}
+              done
+              
+              put_index=$(curl -s ${ADMIN_API_PATH}/${CONTROLLER_STATUS_PATH} | jq -r .commited_index)
+              
+              put_post_diff=$(expr $put_index - $post_index)
+              
+              # we fail here as put should be idempotent
+              if [ "${put_post_diff}" -ge "${CREATE_INDEX_LOWER_BOUND}" ]; then
+                FAILED_MSGS="${FAILED_MSGS}; PUT failed: index is ${put_index}, greater than 2 x ${CREATE_INDEX_LOWER_BOUND}"
+                echo "PUT failed: index is ${put_index}, greater than 1000"
+              fi
```

Comment:
This looks like it's testing Redpanda. Redpanda should have its own test for this.

Reply:
Agreed. This was in the hopes to capture regression, that is past vs future solution, but it seems that will not be resolved in this way and yes that should belong in a duck-tape test anyway.